src/mockserver.js

```diff
@@ -1,9 +1,12 @@
 var config = window.__karma__.config,
-	ui5config = (config && config.openui5) || {},
+	ui5config = (config && config.openui5.config) || {},
 	mockserverConfig = ui5config.mockserver || {};
 
 //workaround for crossroads/mockserver issue
 (function() {
+	if (typeof module === undefined) {
+		var module = '';
```

Comment:
This is wrong. If `module` is `undefined` it should stay `undefined`and should not be made to an emtpy string.